### PR TITLE
Benji/ios 343 add typing animation

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: Shared/Models/MicroAnimation.swift [hunk 1]
```diff
@@ -9,14 +9,8 @@
 import Foundation
 
 enum MicroAnimation: String {
-    case pie = "pie_loader"
-    case scrollDown = "scroll_down"
```
Comment (Benjamindodgson) [inline Shared/Models/MicroAnimation.swift:13]: removing unused animations

File: Shared/UI/View Controllers/NavigationBarViewController.swift [hunk 2]
```diff
@@ -28,9 +27,7 @@ class NavigationBarViewController: ViewController {
         super.initializeViews()
 
         self.view.addSubview(self.blurView)
-        self.animationView.transform = CGAffineTransform(rotationAngle: halfPi * -1)
```
Comment (Benjamindodgson) [inline Shared/UI/View Controllers/NavigationBarViewController.swift:31]: not used.

File: Shared/UI/Views/DisplayableImageView.swift [hunk 3]
```diff
@@ -154,13 +154,7 @@ class DisplayableImageView: View {
                 self.state = .loading
             }
 
-            let data = try await file.retrieveDataInBackground { [weak self] progress in
-                guard let `self` = self else { return }
-                if self.animationView.microAnimation == .pie {
-                    let time = AnimationProgressTime(progress)
```
Comment (Benjamindodgson) [inline Shared/UI/Views/DisplayableImageView.swift:160]: not used

File: iOS/Flows/Conversation/Conversation Header/MemberCell.swift [hunk 4]
```diff
@@ -30,15 +30,23 @@ class MemberCell: CollectionViewManagerCell, ManageableCell {
     var currentItem: Member?
 
     let avatarView = AvatarView()
-    private var animationView = AnimationView.with(animation: .typing)
+    
+    lazy var pulseLayer: CAShapeLayer = {
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Conversation Header/MemberCell.swift:34]: added a pulse

**Review by MartinJibber — APPROVED** [review]

**Pull request closed without merging**